src/EditorFeatures/CSharp/Formatting/CSharpEditorFormattingService.cs

```diff
@@ -271,14 +259,17 @@ private bool IsEndToken(SyntaxToken endToken)
 
         private bool ValidSingleOrMultiCharactersTokenKind(char typedChar, SyntaxKind kind)
         {
-            ImmutableHashSet<SyntaxKind> set;
-            if (!_multiWordsMap.TryGetValue(typedChar, out set))
+            switch (typedChar)
```

Comment:
Can you comment what this code even does? I have no clue what to do if I was asked to modify this.